src/main/scala/is/hail/methods/Aggregators.scala

```diff
@@ -285,6 +285,48 @@ class FractionAggregator(f: (Any) => Any)
   def copy() = new FractionAggregator(f)
 }
 
+class ExistsAggregator(f: (Any) => Any)
+  extends TypedAggregator[Boolean] {
+
+  var exists: Boolean = false
+
+  def result: Boolean = exists
+
+  def seqOp(x: Any) {
+    val r = f(x)
+    if (r != null && r.asInstanceOf[Boolean])
+      exists = true
+  }
+
+  def combOp(agg2: this.type) {
+    if (agg2.exists)
+      exists = true
+  }
+
+  def copy() = new ExistsAggregator(f)
+}
+
+class ForallAggregator(f: (Any) => Any)
```

Comment:
I thought about that but "implementing" one aggregator in terms of another one didn't seem as clear.